### PR TITLE
Update dependencies.

### PR DESCRIPTION
**Pull request by keathley**
Updating our dependencies.

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -49,15 +49,15 @@ defmodule Wallaby.Mixfile do
 
   defp deps do
     [
-      {:httpoison, "~> 0.12"},
+      {:httpoison, "~> 0.13"},
```
Comment (PragTob) [inline mix.exs]: Is there a feature we need to use from the new httpoison? For runtime requirements I'd always try to keep it as the lowest common denominator, I mean we allow 0.13 already with this :)

Comment (keathley) [reply mix.exs]: I don't believe there's anything in there that we need. We can keep it at 0.12 for a while longer.

**Comment by PragTob** [comment]
1.4.5 on 19.3 with Selenium 3 is really... fun and blowing up our tests a lot. Restarted build so hopefully it passes now...

**Comment by keathley** [comment]
@PragTob I'm working on a custom docker image that will hopefully make these builds more reproducible (and maybe even faster?).

**Comment by keathley** [comment]
@aaronrenner does this look ok now?

**Comment by PragTob** [comment]
Well then... :shipit:

**Comment by aaronrenner** [comment]
@keathley About the docker image... before you do that, I'd like to make it so the SeleniumDriver handles starting up and monitoring the selenium server like we do with phantom and chromedriver. Also, once we make it so phantom doesn't start up automatically, we can configure our builds to only install webdriver server (phantom/chrome/selenium) that we need. I know it's annoying right now, but if we can make these changes and get things working reliably, we won't have a docker image to maintain.

**Pull request merged**